platform/common/core/generic/vault/db/helpers.go

```diff
@@ -33,7 +33,7 @@ func OpenMemory() (*db.UnversionedPersistence, error) {
 }
 
 func openMemory[V any](open opener[V]) (V, error) {
-	return open(&mem.Driver{}, "", nil)
+	return open(&mem.Driver{}, "memory", nil)
```

Comment:
Can you use `mem.MemoryPersistence` just to keep track of where we use this constant.

Reply:
yes